### PR TITLE
DDF-3924 Prevents more than one tab from showing at once

### PR DESCRIPTION
**Pull request by andrewkfiedler**
#### What does this PR do?
Prevents the possibility of more than one tab being shown at once in the even that multiple shown events get generated at the same time.

#### Who is reviewing it? 
@roelens8 
@mcalcote 
@mackncheesiest 
@djblue 
@clockard 

#### How should this be tested? (List steps with links to updated documentation)
Go to the admin UI and run either manually attempt to click tabs fast, or use this in the web console.
```$('[data-toggle=tab]').click()```
Verify only one tab gets shown at once.

#### What are the relevant tickets?
[DDF-3924](https://codice.atlassian.net/browse/DDF-3924)

### CODE REVIEW

**Comment by mcalcote** [comment]
Tested the changes locally, fixed!

File: ui/packages/ui/src/main/webapp/components/tab-content/tab-content.view.js [hunk 1]
```diff
@@ -53,6 +54,11 @@ define([
                 }));
             }
         },
+        handleTabHidden: function(id) {
+            if (id === this.model.id) {
+                this.$el.removeClass('active').removeClass('in');
```
Comment (djblue) [inline ui/packages/ui/src/main/webapp/components/tab-content/tab-content.view.js]: Can we combine these like the method call below?

**Comment by andrewkfiedler** [comment]
build now

**Comment by andrewkfiedler** [comment]
build now

**Comment by bdeining** [comment]
build now

**Pull request merged**